datafusion/functions/src/string/bit_length.rs

```diff
@@ -55,7 +58,10 @@ impl Default for BitLengthFunc {
 impl BitLengthFunc {
     pub fn new() -> Self {
         Self {
-            signature: Signature::string(1, Volatility::Immutable),
+            signature: Signature::coercible(
```

Comment:
as a minor comment this new signature is quite a bit of a mouthful compared to `Signature::string`

It isn't clear to me from reading the code / comments when one would prefer to use one over the other (not related to this PR)